### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by jacobtoppm**
Add unit tests for simple functionality of Converters and GoogleDocumentParser

### CODE REVIEW

File: wagtail_content_import/mappers/tests.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        image_converter = ImageConverter('test_block')
+        image_converter.fetch_image = MagicMock(return_value=('content_import_test_image', b"\x89PNG\r\n\x1a\n\x00\x00\x00\rIHDR\x00\x00\x00\x01\x00\x00\x00\x01\x01\x03\x00\x00\x00%\xdbV\xca\x00\x00\x00\x03PLTE\x00\x00\x00\xa7z=\xda\x00\x00\x00\x01tRNS\x00@\xe6\xd8f\x00\x00\x00\nIDAT\x08\xd7c`\x00\x00\x00\x02\x00\x01\xe2!\xbc3\x00\x00\x00\x00IEND\xaeB`\x82"))
+        #one pixel image from https://upload.wikimedia.org/wikipedia/commons/c/ca/1x1.png
+        test_user = User.objects.create_user(username='tester', email='test@test', password='top_secret')
+        converted_element = image_converter({'type': 'image', 'value': 'url'}, user=test_user)
+        Image = get_image_model()
+        self.assertEqual(converted_element[0], 'test_block', "Should be 'test_block'")
+        self.assertIsInstance(converted_element[1], Image)
+        self.assertIn(converted_element[1], Image.objects.filter(title='content_import_test_image'))
+
```
Comment (kaedroho) [inline wagtail_content_import/mappers/tests.py]: Please could you remove this extra whitespace?

File: MANIFEST.in [hunk 2]
```diff
@@ -1,3 +1,4 @@
 recursive-include wagtail_content_import/templates *
 recursive-include wagtail_content_import/pickers/google/templates *
-recursive-include wagtail_content_import/pickers/google/static *
+recursive-include wagtail_content_import/pickers/google/static*
```
Comment (kaedroho) [inline MANIFEST.in]: The asterisk seems to have moved here, not sure if this matters but it no longer matches the previous lines

File: wagtail_content_import/mappers/tests.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        table_converter = TableConverter('test_block')
+        test_table = Table([[Cell('0'), Cell('1')], [Cell('2'), Cell('3')]])
+        test_element = {'type': 'table', 'value': test_table}
+        converted_element = table_converter(test_element)
+        self.assertEqual(converted_element[0], 'test_block', "Should be 'test_block'")
+        self.assertEqual(converted_element[1], {'data': [['0', '1'], ['2', '3']]}, "Should be: {'data': [['0', '1'], ['2', '3']]}")
+
+    def test_image_conversion(self):
+        image_converter = ImageConverter('test_block')
+        image_converter.fetch_image = MagicMock(return_value=('content_import_test_image', b"\x89PNG\r\n\x1a\n\x00\x00\x00\rIHDR\x00\x00\x00\x01\x00\x00\x00\x01\x01\x03\x00\x00\x00%\xdbV\xca\x00\x00\x00\x03PLTE\x00\x00\x00\xa7z=\xda\x00\x00\x00\x01tRNS\x00@\xe6\xd8f\x00\x00\x00\nIDAT\x08\xd7c`\x00\x00\x00\x02\x00\x01\xe2!\xbc3\x00\x00\x00\x00IEND\xaeB`\x82"))
```
Comment (kaedroho) [inline wagtail_content_import/mappers/tests.py]: You could use `wagtail.images.tests.utils.get_test_image_file` to generate a fake image.

See: https://github.com/wagtail/wagtail/blob/master/wagtail/images/tests/utils.py#L11

File: wagtail_content_import/parsers/tests.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
+from django.test import TestCase
+
+from .google import GoogleDocumentParser
+
+import json
```
Comment (kaedroho) [inline wagtail_content_import/parsers/tests.py]: We should put standard library imports at the top

File: wagtail_content_import/mappers/tests.py [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+from wagtail.images import get_image_model
+
+from unittest.mock import MagicMock
+
+from .converters import ImageConverter, RichTextConverter, TextConverter, TableConverter
+from ..parsers.tables import Table, Cell
+
+
+class testConverters(TestCase):
```
Comment (kaedroho) [inline wagtail_content_import/mappers/tests.py]: Class names should start with a capital letter

File: wagtail_content_import/mappers/tests.py [hunk 6]
```diff
@@ -0,0 +1,58 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+
+from wagtail.images import get_image_model
+
+from unittest.mock import MagicMock
```
Comment (kaedroho) [inline wagtail_content_import/mappers/tests.py]: Could you move this import to the top?

File: wagtail_content_import/mappers/tests.py (hunk 3 above)
Comment (jacobtoppm) [reply wagtail_content_import/mappers/tests.py]: Thanks, I've changed to do this

File: wagtail_content_import/mappers/tests.py (hunk 5 above)
Comment (jacobtoppm) [reply wagtail_content_import/mappers/tests.py]: Thanks for spotting!

File: wagtail_content_import/parsers/tests.py (hunk 4 above)
Comment (jacobtoppm) [reply wagtail_content_import/parsers/tests.py]: Thanks, I've fixed.

File: wagtail_content_import/mappers/tests.py (hunk 6 above)
Comment (jacobtoppm) [reply wagtail_content_import/mappers/tests.py]: Thanks, have done!

**Pull request merged**